app/src/main/kotlin/com/intive/picover/common/text/PicoverOutlinedTextField.kt

```diff
@@ -24,8 +24,7 @@ fun PicoverOutlinedTextField(
 	keyboardType: KeyboardType = KeyboardType.Text,
 	imeAction: ImeAction = ImeAction.Done,
 	keyboardActions: KeyboardActions = KeyboardActions(),
-	isError: Boolean = false,
-	errorText: String? = null,
+	@StringRes errorMessageId: Int? = null,
```

Comment:
It should rather take a `String` as a parameter. We don't want to have two parameters for error, this one and for regular `String`

Reply:
I would also go with `String` parameter

Reply:
Done